lib/assessment.js

```diff
@@ -32,13 +42,20 @@ var any = function any(array, callback, thisArg) {
 // made up of the array elements for which invoking
 // the callback returns a truthy value
 var select = function select(array, callback, thisArg) {
+  var newAry = [];
+
+  newAry = array.forEach(callback(currentValue, index, array))
 
+  return newAry;
```

Comment:
You're right that we need to return a new array. How might we make it so that we only fill that new array with elements for which `callback(element)` gives us `true`?